rubygem-mysql2/rubygem-mysql2.spec

```diff
@@ -16,11 +16,9 @@ URL: http://mysql-win.rubyforge.org
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires: %{?scl_prefix_ruby}ruby >= 1.8.6
-Requires: %{?scl_prefix_ruby}rubygems >= 1.8.10
 
 BuildRequires: %{?scl_prefix_ruby}ruby >= 1.8.6
 BuildRequires: %{?scl_prefix_ruby}rubygems >= 1.8.10
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
```

Comment:
You need this for macros to expand, but perhaps remove the version spec from the line above.

<pre>
+ cp -a './%{gem_dir}/build_info' './%{gem_dir}/cache' './%{gem_dir}/doc' './%{gem_dir}/gems' './%{gem_dir}/specifications' '/builddir/build/BUILDROOT/rubygem-mysql2-0.3.19-2.git.229.98d6e60.git.229.98d6e60.el7.x86_64/%{gem_dir}'
cp: target '/builddir/build/BUILDROOT/rubygem-mysql2-0.3.19-2.git.229.98d6e60.git.229.98d6e60.el7.x86_64/%{gem_dir}' is not a directory
</pre>